### DISCUSSION TITLE
[Feature Request]HEVC解码支持

### DISCUSSION THREAD

**Discussion by YiuTerran** [bytedance/xgplayer#841] [General]
感谢贵司在chrome 107+的版本提交了硬解HEVC的代码，希望西瓜播放器能够跟进这个支持，使H.265的视频流更加普及。

**Comment by gemxx** [comment]
> 感谢贵司在chrome 107+的版本提交了硬解HEVC的代码，希望西瓜播放器能够跟进这个支持，使H.265的视频流更加普及

您具体需求是什么呢？HEVC特性探测还是各插件对HEVC的支持？

**Reply by YiuTerran** [reply]
@gemxx 事实上我们在做视频监控这块的工作，目前主流摄像头厂商（海康/大华这些）现在主推的都是H.265，虽然可以手动调整成H.264但是带宽消耗大了很多。
而且现阶段开源播放器中，H265播放器是独立的生态，还没有任何一家web开源播放器同时支持AVC和HEVC，感觉是个空白。

**Reply by YiuTerran** [reply]
另外，我看其他帖子里说浏览器底层支持H.265的话播放器就应该支持，但是实际使用中发现某些号称支持H.265的播放器实际使用中并不能正确的播放码流（比如FLV+H.265，或者TS+H.265），当然这可能和FLV官方并不支持H.265有关。

**Reply by gemxx** [reply]
我们会在今年年底，或者明年初release xgplayer3.x，其中 xgplayer-hls 及 xgplayer-flv都会支持265硬解，请保持关注，感谢使用

**Reply by Hiram-Wong** [reply]
> 我们会在今年年底，或者明年初release xgplayer3.x，其中 xgplayer-hls 及 xgplayer-flv都会支持265硬解，请保持关注，感谢使用

v3正式版预计什么时候发行，以及 xgplayer-hls 对支持h265什么时候支持
